include/constants.h

```diff
@@ -261,9 +278,9 @@ enum dcontrols
          * - mod_pk = kappa_eps = delta_e = 0: circular geometry [x=r.cos(theta), y=r.sin(theta)]
          * - else: stretched circular geometry
          *   [x=(1-kappa_eps)r.cos(theta)-delta_e.r^2, y=(1+kappa_eps)r.sin(theta)] with
-         *     - kappa_eps: Elongation
+         *     - kappa_eps: Elongation (Parameter kappa for the Shafranov geometry, parameters epsilon for the Czarny geometry, is set automatically depending on the geometry)
```

Comment:
```suggestion
         *     - kappa_eps: Elongation (For the Shafranov geometry, the parameter is denoted kappa, for the Czarny geometry, it is epsilon.
```

I propose to recopy parts of my mod_pk suggestions above to the brief description above